controllers/cache_handler_test.go

```diff
@@ -138,11 +138,13 @@ var _ = Describe("Cache Handler verifications", func() {
 			Expect(err).To(BeNil())
 
 			By("ensuring created event was fired")
-			event := findEvent(disruption.CreationTimestamp.Time, v1beta1.Events[v1beta1.EventDisrupted], eventList.Items, targetPodA.Name)
-			Expect(event).ToNot(BeNil())
+			Eventually(func(g Gomega) {
```

Comment:
Why is `g` passed here? Naive question, I don't know what it is (we are not using it in other `Eventually` calls).

Reply:
I took it from the `Eventually` docs:
```

    Eventually(func(g Gomega) {
    	model, err := client.Find(1138)
    	g.Expect(err).NotTo(HaveOccurred())
    	g.Expect(model.Reticulate()).To(Succeed())
    	g.Expect(model.IsReticulated()).To(BeTrue())
    	g.Expect(model.Save()).To(Succeed())
    }).Should(Succeed())
```

Reply:
It's another signature of Gomega:
* https://onsi.github.io/gomega/#category-3-making-assertions-eminem-the-function-passed-into-codeeventuallycode

Eventually can take a value, a func returning a type to perform assertion, or taking gomega as a parameter to perform complex assertion inside the Eventually func body (and eventually returning some values also)